sklearn/model_selection/tests/test_validation.py

```diff
@@ -2399,3 +2399,28 @@ def test_learning_curve_partial_fit_regressors():
 
     # Does not error
     learning_curve(MLPRegressor(), X, y, exploit_incremental_learning=True, cv=2)
+
+
+def test_cross_validate_return_indices(global_random_seed):
+    """Check the behaviour of `return_indices` in `cross_validate`."""
+    X, y = load_iris(return_X_y=True)
+    clf = SVC(gamma="auto")
+    grid = GridSearchCV(clf, param_grid={"C": [1, 10]})
```

Comment:
What is the purpose of using an expensive meta estimator in this test? Testing directly with a dummy classifier would be enough, no?

Reply:
Indeed, I don't recall. I might have only copy pasting a previous test and adapted it. I agree that having nested cross-validation should not impact the behavior here.